docs/WebEditorBasics/SharingExperiences.md

```diff
@@ -0,0 +1,55 @@
+---
+id: SharingExperiences
+title: Sharing Experiences with Others
+---
+
+## Adding Collaborators
+If you would like to share an experience for others to view or edit, you can add them as **Collaborators**.
+
+To add a collaborator, click on the **Share** button in the top right corner of the web editor. Doing so will bring up a modal dialog box in the middle of the screen.
+
+![Modal Dialog](/img/product/editor/sharing/share_button_and_modal.gif)
+
+This modal dialog box will display not only the current collaborators on the project, but also their [permission level](#changing-permissions).
+
+### Inviting Enklu Users
+
+At the top of the dialog box is a search bar, where you can type an individual's email address to add them to your project. If the entered email does have an associated Enklu account, it will display the associated username for you to confirm the correct collaborator. Make sure to only share your experiences with people you trust.
+
+![Collaborator email found](/img/product/editor/sharing/sharing_enklu_users.png)
+
+### Inviting Non-Enklu Users
+
+If a collaborator's email address is not currently associated with an Enklu account, you will be able to send an email invite. This email invite will allow the recipient to create an Enklu account and share your selected experience with them once their account has been created.
+
+![Collaborator email not found](/img/product/editor/sharing/sharing_non_enklu_users.gif)
+
+### Removing a Collaborator
```

Comment:
Not a big deal, but this feels more like an h2 than an h3

Reply:
Re: placeholder.png, that was just the Enklu logo that I used as, well, a placeholder before I had actual gifs and pngs haha. I can remove it tho